cdap-ui/app/cdap/components/Overview/index.js

```diff
@@ -81,16 +127,75 @@ export default class Overview extends Component {
       paginationContainer.scrollTop -= 120;
     }
   }
-  componentDidMount() {
-    this.scrollEntityToView();
+  bindKeyboardShortcuts() {
+    this.mousetrap = new Mousetrap(document);
```

Comment:
Will this cause memory leak? Is there a tear down method required for existing Mousetrap object?